include/mapnik/expression_grammar_x3_def.hpp

```diff
@@ -130,7 +140,7 @@ namespace mapnik { namespace grammar {
 
     auto do_unicode = [] (auto const& ctx)
     {
-        auto const& tr = x3::get<transcoder_tag>(ctx).get();
+        auto const& tr = extract_transcoder(ctx);
```

Comment:
```c++
mapnik::transcoder const& tr = x3::get<transcoder_tag>(ctx);
```

Reply:
yes, fully spelling out returned type works ^ but I prefer helper function still, specially when it used multiple time.